opta/utils/runtee.py

```diff
@@ -0,0 +1,159 @@
+# type: ignore
+# nosec
+"""tee like run implementation."""
+import asyncio
```

Comment:
Was this file found at another source? If so, we probably need to mention that and maybe add license text to the file.